src/cli.rs

```diff
@@ -274,189 +292,239 @@ pub fn handle_cli(cli: &Cli) {
+                    info!("Enter the screen name to confirm the screen deletion: ");
+                    io::stdout().flush().unwrap();
+
+                    let stdin = io::stdin();
+                    let mut user_input = String::new();
+                    match stdin.read_line(&mut user_input) {
+                        Ok(_) => {}
+                        Err(e) => {
+                            error!("Error occurred: {}", e);
                             std::process::exit(1);
                         }
                     }
-                    Err(e) => {
-                        error!("Error occurred: {}", e);
+
+                    if name != user_input.trim() {
+                        error!("The name you entered is incorrect. Aborting.");
                         std::process::exit(1);
                     }
                 }
-                match asset_command.delete(uuid) {
-                    Ok(()) => {
-                        info!("Asset deleted successfully.");
-                        std::process::exit(0);
+                Err(e) => {
+                    error!("Error occurred: {}", e);
+                    std::process::exit(1);
+                }
+            }
+
+            match screen_command.delete(uuid) {
+                Ok(()) => {
+                    info!("Screen deleted successfully.");
+                    std::process::exit(0);
+                }
+                Err(e) => {
+                    error!("Error occurred: {:?}", e);
+                    std::process::exit(1);
+                }
+            }
+        }
+    }
+}
+
+pub fn handle_cli_asset_command(command: &AssetCommands) {
+    let authentication = Authentication::new();
+    match command {
+        AssetCommands::List { json } => {
+            let asset_command = commands::asset::AssetCommand::new(authentication);
+            handle_command_execution_result(asset_command.list(), json);
+        }
+        AssetCommands::Get { uuid, json } => {
+            let asset_command = commands::asset::AssetCommand::new(authentication);
+            handle_command_execution_result(asset_command.get(uuid), json);
+        }
+        AssetCommands::Add { path, title, json } => {
+            let asset_command = commands::asset::AssetCommand::new(authentication);
+            handle_command_execution_result(asset_command.add(path, title), json);
+        }
+        AssetCommands::Delete { uuid } => {
+            let asset_command = commands::asset::AssetCommand::new(authentication);
+            match get_asset_title(uuid, &asset_command) {
+                Ok(title) => {
+                    info!("You are about to delete the asset named \"{}\".  This operation cannot be reversed.", title);
+                    info!("Enter the asset title to confirm the asset deletion: ");
+                    io::stdout().flush().unwrap();
```

Comment:
Do we support flags like `--force`, or `--noinput` to skip confirmation steps?

Reply:
no

Reply:
(but maybe in the future we will add it).